hunt/app/views/animals/edit.html.erb

```diff
@@ -4,7 +4,7 @@
       <div class="card">
         <div class="card-header"><%= 'Edit ' + @animal.name %></div>
         <div class="card-body">
-          <%= render partial: "form", locals: {cancel_url: animal_path} %>
+          <%= render partial: "form", locals: {post: @animal, cancel_url: animal_path} %>
```

Comment:
Looks like these `post` variables got added back in, they don't hurt anything but let's go ahead and take them out.